pkg/render/fluentd.go

```diff
@@ -214,6 +215,13 @@ func (c *fluentdComponent) fluentdNodeName() string {
 	return FluentdNodeName
 }
 
+func (c *fluentdComponent) fluentdMetricsService() string {
```

Comment:
Should be `fluentdMetricsServiceName` to be more clear

Reply:
This should also have a comment explaining why we need / want two names

Reply:
Will do, thanks!